src/components/OverviewCard.jsx

```diff
@@ -39,14 +43,24 @@ const OverviewCard = ({
   const [canShowDropdown, setCanShowDropdown] = useState(false)
   const [canShowFileMoveDropdown, setCanShowFileMoveDropdown] = useState(false)
   const [canShowDeleteWarningModal, setCanShowDeleteWarningModal] = useState(false)
+  const [canShowWarningModal, setCanShowWarningModal] = useState(false)
```

Comment:
Can we rename this hook to `canShowGenericWarningModal`?

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/247/commits/85b8607409d420533e135dc9abda37c9e390c210